smarts/core/smarts.py

```diff
@@ -711,16 +724,10 @@ def neighborhood_vehicles_around_vehicle(self, vehicle, radius=None):
         return [other_states[i] for i in indices]
 
     def vehicle_did_collide(self, vehicle_id):
-        return (
-            len(
-                [
-                    c
-                    for c in self._vehicle_collisions[vehicle_id]
-                    if c.collidee_id != self._ground_bullet_id
-                ]
-            )
-            > 0
-        )
+        for c in self._vehicle_collisions[vehicle_id]:
+            if c.collidee_id != self._ground_bullet_id:
+                return True
+        return False
 
```

Comment:
oops.  This is an unrelated change (very minor optimization) that I accidentally left in.  I think it's harmless, but I'm happy to take it back out of this PR if you think I should.